src/portable/synopsys/dwc2/dwc2_esp32.h

```diff
@@ -34,10 +34,22 @@
 
 #include "esp_intr_alloc.h"
 #include "soc/periph_defs.h"
-//#include "soc/usb_periph.h"
```

Comment:
Please check whether we still need this includes and remove commented-out code

Reply:
I have tried use it, but we need to declare reg base addresses for S2/S3 in esp-idf (`usb_reg.h`, `soc.h` and `reg_base.h`)

    soc/usb_reg.h:20:26: error: 'SOC_DPORT_USB_BASE' undeclared here (not in a function) 
        20 | #define USB_GOTGCTL_REG (SOC_DPORT_USB_BASE + 0x0)


So, for now I think I will leave just the clean base addresses and we can revisit this after making an order in files, I have mentioned in my first sentence. 
And remove the annotated line.